realm/realm-library/src/androidTest/kotlin/io/realm/ManagedSetTester.kt

```diff
@@ -420,6 +421,53 @@ class ManagedSetTester<T : Any>(
         assertSetContainsSet(initializedSet.plus(notPresentValue), set)
     }
 
+    override fun putRequired() {
```

Comment:
Odd name considering what it is actually doing. If not testing this as part of the methods for the individual operations, maybe rename to `requiredConstraints` 🤷